src/styles/components/nodes-grid-view.less

```diff
@@ -78,6 +78,102 @@
         }
 
         &.service-color-8 {
+          background-color: rgba(219, 127, 103, 1);
+        }
+
+        &.service-color-9 {
+          background-color: rgba(219, 190, 161, 1);
+        }
+
+        &.service-color-10 {
+          background-color: rgba(163, 123, 115, 1);
+        }
+
+        &.service-color-11 {
+          background-color: rgba(63, 41, 43, 1);
+        }
+
+        &.service-color-12 {
+          background-color: rgba(211, 79, 115, 1);
+        }
+
+        &.service-color-13 {
+          background-color: rgba(72, 64, 65, 1);
+        }
+
+        &.service-color-14 {
+          background-color: rgba(67, 67, 113, 1);
+        }
+
+        &.service-color-15 {
+          background-color: rgba(121, 174, 163, 1);
+        }
+
+        &.service-color-16 {
+          background-color: rgba(112, 238, 156, 1);
+        }
+
+        &.service-color-17 {
+          background-color: rgba(181, 244, 74, 1);
+        }
+
+        &.service-color-18 {
+          background-color: rgba(173, 52, 62, 1);
+        }
+
+        &.service-color-19 {
+          background-color: rgba(71, 71, 71, 1);
+        }
+
+        &.service-color-20 {
+          background-color: rgba(242, 175, 41, 1);
+        }
+
+        &.service-color-21 {
+          background-color: rgba(224, 224, 206, 1);
+        }
+
+        &.service-color-22 {
+          background-color: rgba(252, 170, 103, 1);
+        }
+
+        &.service-color-23 {
+          background-color: rgba(176, 65, 62, 1);
+        }
+
+        &.service-color-24 {
+          background-color: rgba(255, 255, 199, 1);
+        }
+
+        &.service-color-25 {
+          background-color: rgba(84, 134, 135, 1);
+        }
+
+        &.service-color-26 {
+          background-color: rgba(71, 51, 53, 1);
+        }
+
+        &.service-color-27 {
+          background-color: rgba(72, 64, 65, 1);
+        }
+
+        &.service-color-28 {
+          background-color: rgba(67, 67, 113, 1);
+        }
+
+        &.service-color-29 {
+          background-color: rgba(121, 174, 163, 1);
+        }
+
+        &.service-color-30 {
+          background-color: rgba(112, 238, 156, 1);
+        }
+
+        &.service-color-31 {
+          background-color: rgba(181, 244, 74, 1);
+        }
```

Comment:
Just curious, where are these colors coming from?

Also, we don't need to use `rgba` because the alpha channel is being set to 1. But I don't think it really makes a difference, so feel free to leave as-is.